docs/l3/api_flow.md

```diff
@@ -0,0 +1,413 @@
+# Prerequisites (dscp-api-identity-service)
+
+## Setting node aliases 
+
+The values set for each persona are your choice, they should provide a recognisable value in response bodies.
+
+Here we have went with:
+
+- “Member A” to represent an order
+
+- “Member B” to represent a capacity
+
+- “Optimiser” to represent an optimiser
+
+1. For each node, set node address `5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY` to “Member A” i.e. Order, using `PUT /members/{address}`
+
+2. For each node, set node address `5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty` to “Member B” i.e. Capacity, using `PUT /members/{address}`
```

Comment:
same here, MemberA =/= an order, they are the persona who makes the order

Reply:
```suggestion
1. For each node, set node address `5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY` to “Member A” i.e. the persona placing an Order, using `PUT /members/{address}`

2. For each node, set node address `5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty` to “Member B” i.e. the persona offering Capacity, using `PUT /members/{address}`
```